app/api/server/v1/email-inbox.js

```diff
@@ -0,0 +1,142 @@
+import { check, Match } from 'meteor/check';
+
+import { API } from '../api';
+import { findEmailInboxes, findOneEmailInbox, insertOneOrUpdateEmailInbox } from '../lib/emailInbox';
+import { hasPermission } from '../../../authorization/server/functions/hasPermission';
+import { EmailInbox } from '../../../models';
+import Users from '../../../models/server/models/Users';
+import { sendTestEmailToInbox } from '../../../../server/features/EmailInbox/EmailInbox_Outgoing';
+
+API.v1.addRoute('email-inbox.list', { authRequired: true }, {
+	get() {
+		try {
```

Comment:
These try catches are not necessary, if an exception happens inside a route it will return an API failure with the error already. https://github.com/RocketChat/Rocket.Chat/blob/develop/app/api/server/api.js#L403